tests/test_jemx_plugin.py

```diff
@@ -143,3 +143,49 @@ def test_jemx_products(dispatcher_long_living_fixture, product_type):
 # def validate_product(product_type, product):
 #     if product_type == "jemx_lc":
 #         print(product.show())
+
+
+def test_jemx_lc_source_name_formatting(dispatcher_long_living_fixture):
```

Comment:
did you mean to add it? This is not going to work here. At least, it should annotated/

Reply:
nope, something that has remained there by mistake. Sorry for that